src/VisualStudio/Core/Def/Implementation/ProjectSystem/FileChangeTracker.cs

```diff
@@ -75,7 +76,7 @@ public bool PreviousCallToStartFileChangeHasAsynchronouslyCompleted
             get
             {
                 var cookie = _fileChangeCookie;
-                return cookie != s_none && cookie.IsValueCreated;
+                return cookie != s_none && cookie.TryGetValue(out _);
             }
```

Comment:
the very nature of this function scares me.  seems so racy.

Reply:
Yep.